ros/src/tl_detector/tl_detector.py

```diff
@@ -146,13 +147,15 @@ def get_closest_waypoint(self, pose):
             int: index of the closest waypoint in self.waypoints
 
         """
+        index = 0
         distances = []
+        if not self.waypoints is None:
```

Comment:
u can also put the checker in the caller, e.g https://github.com/jiangjin/CarND-Capstone/blob/0d59fd7996c11248e5964a768e96d1c2856211c6/ros/src/tl_detector/tl_detector.py#L193